lib/importproject.cpp

```diff
@@ -311,7 +311,11 @@ void ImportProject::fsParseCommand(FileSettings& fs, const std::string& command)
                 fs.includePaths.push_back(std::move(i));
         } else if (F=='s' && startsWith(fval,"td")) {
             ++pos;
-            fs.standard = readUntil(command, &pos, " ");
+            if (startsWith(fval, "td:")) {
+                fs.standard = fval.substr(3);
+            } else {
+                fs.standard = readUntil(command, &pos, " ");
```

Comment:
hmm this is matching the old code. but it does not seem to me that it would handle a gcc command like this right?

    gcc -std=c99 -c 1.c